src/test/osd/TestOSDMap.cc

```diff
@@ -2280,6 +2310,139 @@ TEST_F(OSDMapTest, blocklisting_everything) {
   }
 }
 
+TEST_F(OSDMapTest, ReadBalanceScore1) {
```

Comment:
I would make the names of the test more descriptive as to what the tests are doing.